wbooks/src/app/screens/Library/components/Book/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Image, Text, View } from 'react-native';
+import { Book as BookInterface } from '@interfaces/book';
+import styles from './styles';
+
+interface Props {
+    book: BookInterface;
+}
+
+function Book ({ book: { title, author, imageUrl } }: Props) {
+
+    return (
+        <View style={styles.bookCard}>
+           <View style={styles.containerBook}>
+                <Image
+                    style={styles.imgBook}
+                    source={ { uri: (imageUrl) ? `${imageUrl}` : 'https://reactnative.dev/img/tiny_logo.png' } }
```

Comment:
This parenthesis are unnecesary 😄 
```suggestion
                    source={ { uri: imageUrl ? `${imageUrl}` : 'https://reactnative.dev/img/tiny_logo.png' } }
```